linter/lintfuncs.go

```diff
@@ -11,24 +11,28 @@ const (
 	ThisUpdateLimit = "96h" // 4 days
 )
 
+// LintProducedAtDate checks that an OCSP Response ProducedAt date is no more than ProducedAtLimit in the past
+// Source: Apple Lint 03
 func LintProducedAtDate(resp *ocsp.Response) error {
 	limit, err := time.ParseDuration(ProducedAtLimit)
 	if err != nil {
 		return err
 	}
 	if time.Since(resp.ProducedAt) > limit {
-		return fmt.Errorf("OCSP Response producedAt date is more than %s in the past", ProducedAtLimit)
+		return fmt.Errorf("OCSP Response producedAt date, %s, is more than %s in the past", resp.ProducedAt, ProducedAtLimit)
```

Comment:
Really minor, but the use of commas here seem a bit weird. I might use single quotes.

`"OCSP Response producedAt date '%s' is more than %s in the past"`. In python I'd use %r but I don't know if there is an equivalent in Golang.